utils.go

```diff
@@ -35,3 +48,87 @@ func getFileContents(file string) (string, error) {
 
 	return string(bytes), nil
 }
+
+func getKernelVersion() (string, error) {
```

Comment:
I saw that, but I did it manually to ensure we can test it by subverting the `/proc/version` file.